precompile/contracts/warp/config.go

```diff
@@ -47,22 +47,24 @@ var (
 // adds specific configuration for Warp.
 type Config struct {
 	precompileconfig.Upgrade
-	QuorumNumerator uint64 `json:"quorumNumerator"`
+	QuorumNumerator              uint64 `json:"quorumNumerator"`
+	RequirePrimaryNetworkSigners bool   `json:"requirePrimaryNetworkSigners"`
```

Comment:
We can make this a different type, just need to make sure it serializes as a bool, if readability or passing inline `true`/`false` is a concern.